google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py

```diff
@@ -59,6 +59,27 @@ def generate_ancestry_path(full_name):
     return ancestry_path
 
 
+def convert_data_to_cai_asset(primary_key, resource, resource_type):
+    """Convert data to CAI formatted fields to be used by Config Validator.
```

Comment:
nit: please make this tweak to be more clear.
```
"""Convert non-CAI data to CAI formatted fields to be used by Config Validator.

This conversions is necessary because Config Validator can only process data in CAI data structure.
```

Reply:
Done!